### ISSUE TITLE
Implement first iteration of Button

### ISSUE THREAD

**Issue by dafn** [FHIDev/Fhi.Designsystem#85]
Figma: [Button](https://www.figma.com/design/VgQG6XeEbFOmHIrxyAOACR/FHI-Designsystem?node-id=46-6360&p=f)

ref: https://github.com/FHIDev/Fhi.Designsystem/issues/23

acceptance criteria:
- It should be implemented in accordance with the designs in Figma
- It should be accessable from the NPM package and CDN
- It should have automated unit and accessibility tests with appropriate coverage
- It should have a story with an introduction, usage instructions, implementation examples and action interactivity
- It should conform to the Web standard on inputs/outputs.
  - e.g form interactions
- It should have appropriate component level design tokens that refrences semantic, and if appropriate, primitive level tokens.

**Comment by proand** [comment]
@Masuso er det mulig å få samlet alt under Docs slik som her: https://next.storybook.designsystemet.no/?path=/docs/komponenter-button--docs, og ikke to menypunkt som her:

![Image](https://github.com/user-attachments/assets/ecdaea2f-2b0d-4fdb-867a-80fe84cceca0)

Også ok med kun "Button", og ikke "fhi-button" ut til publikum synes jeg...

**Comment by Masuso** [comment]
> [@Masuso](https://github.com/Masuso) er det mulig å få samlet alt under Docs slik som her: https://next.storybook.designsystemet.no/?path=/docs/komponenter-button--docs, og ikke to menypunkt som her:
> 
> ![Image](https://github.com/user-attachments/assets/ecdaea2f-2b0d-4fdb-867a-80fe84cceca0)
> 
> Også ok med kun "Button", og ikke "fhi-button" ut til publikum synes jeg...

Det ligger en løsning på det i den nyeste commiten : )
